hack/verify-description.sh

```diff
@@ -45,25 +45,38 @@ find_files() {
         -o -wholename './target' \
         -o -wholename '*/third_party/*' \
         -o -wholename '*/vendor/*' \
+        -o -wholename './pkg/*' \
       \) -prune \
     \) \
-    \( -wholename '*pkg/apis/*/v*/types.go' \
-       -o -wholename '*pkg/api/unversioned/types.go' \
-    \)
+    -wholename './staging/src/k8s.io/api/*/v*/types.go'
```

Comment:
we also have API definitions in `staging/src/k8s.io/kube-aggregator/pkg/apis/*/v*/types.go` and `staging/src/k8s.io/apiextensions-apiserver/pkg/apis/*/v*/types.go`

Reply:
done :+1: